### DISCUSSION TITLE
add progressive series and green coloring completed series

### DISCUSSION THREAD

**Discussion by clafalco** [wger-project/flutter#364] [Ideas]
HI!!! it could be useful to visualize the progress of the series done (example: 1/4 - 2/4 - 3/4 etc.) and the color change of the daily series completed.
example:
![Screenshot_2023-03-16-18-47-00-654_de wger flutter jpg](https://user-images.githubusercontent.com/122402237/225710098-bed2e0f1-2f04-43d8-9096-d792058fb51a.svg)

**Comment by rolandgeider** [comment]
You mean like 1st exercise of 4? I see what you mean. There is the progrss bar at the bottom, but that one counts all the individual sets

**Reply by clafalco** [reply]
ok, maybe seeing the numbers it's more immediate to understand where you're at.
what do you think instead, to highlight the completed series?

**Reply by rolandgeider** [reply]
perhaps we could rotate between different views when clicking on the progress bar? General progress in sets / exercises, current set, etc.

**Comment by clafalco** [comment]
Yes, good idea

**Comment by eHonnef** [comment]
Just a few more suggestions, when I open the burger menu on the top right show an overview of the progress. Currently I have to remember which exercise I have to reorder because someone was using the machine and I have to come back to it.

So the idea would be: keep the list style, but mark as green if done and the overview amount of sets. E.g.:

- Squats  --  0/3
- Leg curl -- 2/3

When I'm doing the exercise I have no overview on how many sets are left, if it is a 3,4 or 5 sets exercise.

Also, in the exercise summary would be nice to have an overview of the planned, a simple list with the sets:

- set 1: 10 reps
- set 2: 12 reps
- set 3: 8 reps

**Reply by rolandgeider** [reply]
good idea, I moved this to #958
